internal/worker/uniter/remotestate/watcher_test.go

```diff
@@ -246,7 +246,6 @@ func (s *WatcherSuiteIAAS) TestInitialSnapshot(c *gc.C) {
 		Relations:               map[int]remotestate.RelationSnapshot{},
 		Storage:                 map[names.StorageTag]remotestate.StorageSnapshot{},
 		ActionChanged:           map[string]int{},
-		UpgradeMachineStatus:    model.UpgradeSeriesNotStarted,
```

Comment:
More to remove in this test file:

* ./internal/worker/uniter/remotestate/watcher_test.go:285:	if s.uniterClient.unit.upgradeSeriesWatcher != nil {
* ./internal/worker/uniter/remotestate/watcher_test.go:286:		s.uniterClient.unit.upgradeSeriesWatcher.changes <- struct{}{}
* ./internal/worker/uniter/remotestate/watcher_test.go:319:		s.uniterClient.unit.upgradeSeriesWatcher.changes <- struct{}{}
* ./internal/worker/uniter/remotestate/watcher_test.go:546:		s.uniterClient.unit.upgradeSeriesWatcher.changes <- struct{}{}
* ./internal/worker/uniter/remotestate/watcher_test.go:1129:		s.uniterClient.unit.upgradeSeriesWatcher.changes <- struct{}{}